src/landing/LandingPage.js

```diff
@@ -1,14 +1,91 @@
 import React, { useState } from 'react';
+import { gql, useLazyQuery, useMutation } from '@apollo/client';
 import {
   Paper, Tab, Tabs,
 } from '@material-ui/core';
 import LoginForm from './LoginForm';
 import './styles.css';
 import RegisterForm from './RegisterForm';
 
+const GET_USERS = gql`
```

Comment:
Not a change request but you may create a file to hold the GQL queries/mutation. Could be named "sources", for example.

Reply:
added "sources.js" file with the queries and mutations